BusinessLogicLayer/UnitOfWork .cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using DataAccessLayer.DataBaseModels;
+using DataAccessLayer.Repositories.Implementation;
+
+namespace BusinessLogicLayer
+{
+    public class UnitOfWork : IDisposable
+    {
+        private LanguageSkillsDBContext _db = new LanguageSkillsDBContext();
+        private CategoryRepository _categoryRepository;
+        private CategoryTranslationRepository _categoryTranslationRepository;
+
+        public CategoryRepository Categories
+        {
+            get
+            {
+                if (_categoryRepository == null)
+                    _categoryRepository = new CategoryRepository(_db);
+                return _categoryRepository;
+            }
+        }
+
+        public CategoryTranslationRepository CategoryTranslations
+        {
+            get
+            {
+                if (_categoryTranslationRepository == null)
+                    _categoryTranslationRepository = new CategoryTranslationRepository(_db);
+                return _categoryTranslationRepository;
+            }
+        }
+
+        public void Save()
```

Comment:
Not good to spread DB logic to two levels - much harder to debug and maintain the code.
Please move "_db.SaveChanges" and other DB-related code to DAL.
as the result of ANY DAL method - DB should return the requested data or be updated with changes.